test-workloads/src/bin/guestvm.rs

```diff
@@ -49,17 +49,136 @@ pub fn alloc_error(_layout: Layout) -> ! {
 /// Test `CertReq` encoded as ASN.1 DER
 const TEST_CSR: &[u8] = include_bytes!("test-ed25519.der");
 
-fn test_attestation(csr_addr: u64) {
```

Comment:
Is the removal of the test_attesation function intentional, or a rebase typo?

Reply:
rebase typo. will fix